docs/sources/github/github-copilot.yaml

```diff
@@ -480,7 +480,10 @@ endpoints:
       - !<tokenize>
         jsonPaths:
           - "$..slug"
-  - pathTemplate: "/orgs/{org}/audit-log"
+  - pathRegex: "^/orgs/[^/]+/audit-log\\?(?=[A-Za-z0-9=&%+:\\._\\-]*phrase=action:copilot(?:\\\
```

Comment:
this is better accomplished with `ParameterSchema` constraints on the query parameters. see: https://docs.worklytics.co/psoxy/configuration/api-data-sanitization#path-query-parameter-schemas

I think will be more readable and easier to validate. Leave `pathTemplate` as is. add `phrase` as `queryParameterSchema`

Reply:
Thanks; spent a while with this... Finally I had to use both, because agree that with query parameters the validation is easy to understand, but I also need that `phrase` and `include` parameters are always present when audit log is invoked, so I've supported that through regex.
We probably can implement something like *mandatoryParameters* or mark as part of *allowedParameters* the ones are mandatories, but I think this is the first case we have wth this and not sure if worth to do it right now.